### PR TITLE
[102] global search template

### PR DESCRIPTION
**Pull request by TrebuhD**
image

- Added search box in top right of website (needs some styling)
- Added new route - `/search/<query>`
- Added `search-view` and a mock response (copied from table view)
- Made the `vaadin-tabs` component somewhat responsive.
- Added `default-passive-events` package to fix a recurring problem in polymer / lit-element.
- Refactored `index.html` and created the `app-layout` component to hook up search input to `lit-element`.

### CODE REVIEW

File: src/views/searchview/search-view.js [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    await this.fetchData();
+  }
+
+  updated(_changedProperties) {
+    super.updated(_changedProperties);
+
+    _changedProperties.forEach(async (oldValue, propName) => {
+      if (
+        [
+          // TODO: uncomment this after filters are added
```
Comment (ayya-vimala) [inline src/views/searchview/search-view.js]: Filters are not needed so can go.

File: src/views/searchview/search-view.js [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+      <data-view-header
+        .score="${this.score}"
+        .quoteLength="${this.quoteLength}"
+        .cooccurance="${this.cooccurance}"
+        .limitCollection="${this.limitCollection}"
+        .fileName="${this.searchQuery}"
+        .language="${this.lang}"
+      ></data-view-header>
+
+      <search-view-list
```
Comment (ayya-vimala) [inline src/views/searchview/search-view.js]: Remove all those references to filters. They have nothing to do with anything.

**Comment by TrebuhD** [comment]
@ayya-vimala this now has changes from master (new design). Most of your gitter comments about the new design have been resolved. If you can approve this, we can merge it.

**Comment by ayya-vimala** [comment]
@TrebuhD : I've made a few css changes but not entirely sure how to properly change those components. Have a look and if okay with you, please merge.

**Comment by TrebuhD** [comment]
@ayya-vimala @sebastian-nehrdich the search page issue should be fixed. I also made some impovements to the main layout, added consistent paddings, etc.

After this is merged, I'll start working on the filter bar.

**Pull request merged**